src/api/zoops.ts

```diff
@@ -33,4 +33,29 @@ zoopsRouter.post("/", async (req, res, next): Promise<void> => {
     }
 })
 
+// DELETE /api/zoops/:id
+// TODO: Add auth middleware to check if user is logged in once it is available from merging of login endpoint branch
+zoopsRouter.delete(`/:id`, async (req, res, next) => {
+    const userId = Number(req.headers.id);
```

Comment:
Do not store the user ID in the headers - there's no need, we can just get it from the auth token and store it on req.user instead

```suggestion
    const userId = req.user.id
```

Reply:
I noticed a TS error when I made this change. It's because `user` is not a native property of Request. I assume that will resolve once I use jsonwebtoken to assign the user data to `req.user`, right? Should I commit this change even with the TS error and just make a TODO about utilizing `req.user` with middleware?

Reply:
Oh interesting - that's a good point, I've mostly seen this kind of extension done not in TS. I wonder what is the best way to extend the req by adding new data now that we're in TS. You may need to create a new type that extends the Request type (eg. RequestWithUser, which could be defined: `type RequestWithUser = Request & {user: User}`)

Reply:
This may also have some guidance: https://blog.logrocket.com/extend-express-request-object-typescript/

Reply:
Another possible approach: https://stackoverflow.com/questions/37377731/extend-express-request-object-using-typescript

Reply:
I'll give this a whirl tomorrow. This also addresses the comments I had in the auth file PR.

Reply:
Was able to extend `Request` and add user to it.